papertrail-cli.js

```diff
@@ -54,7 +60,12 @@ if (argv._.length === 0) {
   return;
 }
 const token = argv.token;
-const search = argv._.length > 1 ? argv._.join(' ') : `'${argv._[0]}'`;
+let search;
+if (argv.exclude) {
+  search = encodeURI(argv._.length > 1 ? `${argv._.join(' AND ')} -"${argv.exclude}"` : `${argv._[0]} -"${argv.exclude}"`);
```

Comment:
I don't think you need all of the ANDs.  You should be able to always join argv._ with `' '` and then add on exclude if its there